### ISSUE TITLE
Undocumented Connections

### ISSUE THREAD

**Issue by mattdog1000000** [lambertjamesd/n64-usb#1]
Are there any requirements for the CH375B VCC pin or the N64 controller 3.3v and ground wires?
(Apologies if these should be incredibly obvious.)

**Comment by lambertjamesd** [comment]
No, I should have documented those. The CH375B should be attached to 5v, the controller 3.3v is unused and tied to nothing. All grounds from the controller wire, CH375B and the Arduino should all be tied together.
